Source/IntegrationTests/TestFiles/LitTests/LitTest/cli/measure-complexity.dfy.expect

```diff
@@ -0,0 +1,30 @@
+Starting verification of iteration 1/1 with seed 0
+TestFiles/LitTests/LitTest/cli/measure-complexity.dfy(5,18): Error: assertion might not hold
+Verification task on line 8 in file measure-complexity.dfy consumed 9984 resources
```

Comment:
I think these lines will be fragile (and might cause the nightly to fail, since Z3 tends to have different resource use on different platforms). Could you move it from an `.expect` file to `CHECK: ` directives in the source file, with wildcards for the actual numbers?